omero/sysadmins/unix/server-installation.txt

```diff
@@ -283,6 +284,17 @@ latest release is available from the :zeroc:`ZeroC website
     ZeroC_ Ice can always be built from source code for specific
     platforms if a binary package is not available.
 
+.. note::
+    With Ice 3.6, the Python bindings are provided separately.  If
+    your package manager does not provide Ice python packages, run::
+
+        pip install zeroc-ice
```

Comment:
This is displayed in sphinx with yellow/green side bars. Doesn't bother me, @hflynn ?